srcopsmetrics/report.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+# SrcOpsMetrics
+# Copyright (C) 2020 Dominik Tuchyna
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Report page running on Dash."""
+
+import os
+from typing import Dict
+
+import dash
+import dash_core_components as dcc
+import dash_html_components as html
+import plotly.graph_objects as go
+from pbr.packaging import append_text_list
```

Comment:
what is pbr?

Reply:
removed
I use an auto import tool in vscode and it probably added this...